src/styles/index.scss

```diff
@@ -1,3 +1,15 @@
-body {
-  margin: 0;
+@import './utils/variables';
+@import './header';
+@import './card/index';
+
+.cards {
+  display: grid;
+  grid-template-columns: minmax(200px, 200px);
```

Comment:
The use of 'minmax(200px, 200px)' is redundant here. It effectively sets a fixed size of 200px, which could be simplified to just '200px'.